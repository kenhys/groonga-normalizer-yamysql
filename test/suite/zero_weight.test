register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "aﾞあﾞz" WITH_CHECKS|WITH_TYPES
