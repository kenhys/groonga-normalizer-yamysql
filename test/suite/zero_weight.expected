register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLUnicodeCI "aﾞあﾞz" WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "AあZ",
    "types": [
      "alpha",
      "hiragana",
      "alpha"
    ],
    "checks": [
      1,
      6,
      0,
      0,
      4
    ]
  }
]
